Algorithm.Python/QuantConnect.Algorithm.Python.csproj

```diff
@@ -160,6 +160,9 @@
       <HintPath>..\packages\QuantConnect.pythonnet.1.0.5.7\lib\Python.Runtime.dll</HintPath>
     </Reference>
   </ItemGroup>
+  <ItemGroup>
+    <None Include="GuaranteedVwapRegressionAlgorithm.py" />
```

Comment:
@AlexCatarino -- do these need to be marked as copy if newer?

Reply:
No, since we use the config.json to get the scripts from `../../Lean/Algorithm.Python/`.